cli/main.go

```diff
@@ -307,7 +304,7 @@ func beforeSubcommands(c *cli.Context) error {
 
 	katautils.SetConfigOptions(name, defaultRuntimeConfiguration, defaultSysConfRuntimeConfiguration)
 
-	configFile, runtimeConfig, tracing, err = katautils.LoadConfiguration(c.GlobalString(configFilePathOption), ignoreLogging, false)
```

Comment:
Since the mainly tracing functions also had been removed into katautils pkg, thus the tracing trigger had been dealt in katautils pkg, and there was no need to return it here.